src/main/resources/config/application-prod.yml

```diff
@@ -132,4 +132,3 @@ info:
             - tutor_assessment_tour: 'Patterns in Software Engineering'
     contact: artemis.in@tum.de                                                                  #default value, can be overridden on the server
     imprint: https://ase.in.tum.de/lehrstuhl_1/component/content/article/179-imprint            #default value, can be overridden on the server
-    test-server: false  # false --> production, true --> test server, --> empty == local
```

Comment:
Is that removal intended?

Reply:
Nope.. Will re-add in the next commit

Thanks for noticing :+1: